recipes/sphinx-autoapi/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "sphinx-autoapi" %}
+{% set version = "0.5.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 45e8fea98ce179adee4b0502eefe7dfeb44f11c8b45391d7b8a6dde15c48ebd1
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - astroid
+    - pyyaml
+    - wheel ==0.24.0
+    - sphinx
+    - sphinxcontrib-golangdomain
+    - sphinxcontrib-dotnetdomain
+    - unidecode
+
+test:
+  requires:
+    - pydocstyle
+    - salvus_mesher_lite  # [osx]
```

Comment:
Can I use `# [osx]` here, even though I'm trying to build this as `noarch: python`?

Reply:
Given it is in the test section, think we are ok. It's not going to impact the final package (just the build steps). Do you mind raising this point to conda-smithy so we can look at adjust the linter to accept this?